src/index.html

```diff
@@ -4,93 +4,93 @@
     </div>
   </div>
 </section>
 
-<section id="our-work" class="our-work usa-grid">
-  <div class="usa-width-one-whole">
-    <h2>Our work</h2>
-    <div class="page-width-70">
+<section id="our-work" class="our-work ">
+
+  <div class="usa-grid">
+    <div class="usa-width-two-thirds usa-offset-one-sixth">
+      <div class="section-rule"></div>
+      <h2>Our work</h2>
       <p>The Department of Defense defends and protects our nation's military and civilians across the globe. However, complex systems and outdated tools can hinder this mission. We use design and technology to improve government services, strengthen national defense, and care for military members and their families. Explore some of our projects below.</p>
     </div>
   </div>
 
-  <div class="dds-grid">
+  <div class="usa-grid">
   {% for project in site.data.projects %}
-    <div class="project dds-default-item">
+    <div class="project dds-default-item usa-width-one-third">
       <img src="{{ project.img_url }}" alt="{{ project.img_description }}">
       <h3>{{ project.title }}</h3>
-      {{ project.project_description }}
+      <p>{{ project.project_description }}</p>
     </div>
   {% endfor %}
   </div>
 </section>
 
-<section id="our-team" class="staff-members usa-grid">
-  <div class="usa-width-one-whole">
-    <h2>Our team</h2>
-    <div class="page-width-70">
+<section id="our-team" class="staff-members">
+  <div class="usa-grid">
+    <div class="usa-width-two-thirds usa-offset-one-sixth">
+      <div class="section-rule"></div>
+      <h2>Our team</h2>
       <p>The Defense Digital Service is a SWAT team of nerds working to improve technology across the Department of Defense. We hire world-class designers, engineers, product managers, and bureaucracy hackers from diverse backgrounds to work on high-impact projects at the Pentagon and across the globe. Team members develop code, manage technical projects or product releases, and rewire outdated IT approaches.</p>
     </div>
   </div>
 </section>
 
-<section id="join" class="join-us usa-grid">
+<section id="join" class="join-us">
   <div class="usa-grid">
-    <div class="join-usds usa-width-one-whole">
+    <div class="usa-width-two-thirds usa-offset-one-sixth">
+      <div class="section-rule"></div>
       <h2>Apply to USDS</h2>
-      <div class="page-width-70">
-        <p>The best of technology. The best of government—and we need you.</p>
-        <p>The Defense Digital Service is an agency team of the U.S. Digital Service. Applicants should apply through USDS and note their interest in working for DDS.</p>
-      </div>
-
-      <a class="join-button" href="https://www.usds.gov/apply">Apply Now</a>
+      <p>The best of technology. The best of government— and we need you.</p>
```

Comment:
Remove space between em dash and "and we need you."

Reply:
✔️